vllm/model_executor/models/aria.py

```diff
@@ -50,6 +51,50 @@ class AriaImagePixelInputs(TypedDict):
     """
 
 
+class AriaVisionTransformer(Idefics3VisionTransformer):
+
+    def __init__(
+        self,
+        config: Idefics2VisionConfig,
+        quant_config: Optional[QuantizationConfig] = None,
+        prefix: str = "",
+    ) -> None:
+        super().__init__(config, quant_config, prefix)
+        self.post_layernorm = nn.Identity()
```

Comment:
Aria model doesn't include the post_layernorm layer, see the discussion at https://github.com/vllm-project/vllm/issues/12241#issuecomment-2605938964

Reply:
done